packages/browser/jest.setup.js

```diff
@@ -4,6 +4,7 @@
 const crypto = require('crypto');
 
 const { location } = require('jest-location-mock');
+const fetch = require('node-fetch');
```

Comment:
Do we still need `fetch` function in browser test? I thought the request related tests are located in `@logto/client`?

Reply:
No, they are located in `browser`, in `client`, we use `jest.fn` to test.